### DISCUSSION TITLE
Updating icon on q-btn-toggle on select

### DISCUSSION THREAD

**Discussion by readysetsneak** [quasarframework/quasar#13170] [General - Components / Directives / etc] [answered]
Hi, I'm trying to use q-btn-toggle, with icon property, and want the icon property to update when a button is selected and un-selected.. I'm having trouble figuring out how to control this option from a method or how to create data binding against the select state of the button? 

Here's a code pen link: https://codepen.io/readysetsneak/pen/jOYvvxK

I'm really new to Vue and Quasar, and still going through tutorials/videos, so please bear with me.

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/milospaunovic/pen/VwyGqWL?editors=1111](https://codepen.io/milospaunovic/pen/VwyGqWL?editors=1111)

**Comment by readysetsneak** [comment]
amazing! thank you! Do you have any resources you would suggest to getting started with Vue and Quasar?

**Reply by MilosPaunovic** [reply]
Official docs for both are pretty good - so probably a good start:

[https://vuejs.org/](https://vuejs.org/)
[https://quasar.dev/](https://quasar.dev/)

There are also great video tutorials for Quasar, if that's your way to go:
[https://quasar.dev/video-tutorials](https://quasar.dev/video-tutorials)
